aerofiles/seeyou/reader.py

```diff
@@ -68,6 +81,23 @@ def decode_waypoint(self, fields):
             'description': self.decode_description(fields[10]),
         }
 
+    def decode_task_info(self, tasks, fields):
+        if fields[0] == 'Options':
+            tasks[-1]['Options'] = self.decode_task_options(fields)
+        elif fields[0].startswith('ObsZone'):
+            tasks[-1]['ObsZones'].append(self.decode_task_ObsZone(fields))
+        else:
+            tasks.append(
+                {
+                    'name': self.decode_task_name(fields),
+                    'waypoints': self.decode_task_waypoints(fields),
+                    'Options': None,
+                    'ObsZones': []
```

Comment:
main reason to use capital letters was to match the strings and to simplify expressions like the following (covering three cases in one):
```
elif field.split("=")[0] in ["WpDis", "MinDis", "RandomOrder"]:            # boolean
                task_options[field.split("=")[0]] = field.split("=")[1] == "True"
```
a possible solutions is to convert CamelCase to snake_case. what do you think about introducing the following general converter function:
http://stackoverflow.com/questions/1175208/elegant-python-function-to-convert-camelcase-to-snake-case